packages/tools/src/store/ToolGroupManager/ToolGroup.ts

```diff
@@ -326,7 +327,7 @@ export default class ToolGroup implements IToolGroup {
     const bindingsToUse = [...prevBindings, ...newBindings].reduce(
       (unique, binding) => {
         if (
-          !unique.some((obj) => obj.mouseButton === binding.mouseButton) &&
+          !unique.some((obj) => hasSameBinding(obj, binding)) &&
```

Comment:
Thanks for the contribution.
Can you please edit the comment above this reducer to say it allows duplicated mouse button but not both mouse and modifier?

Reply:
Thank.
I added the comment you asked for.